DSCResource.Tests

```diff
@@ -0,0 +1 @@
+Subproject commit b367d4beaaa93c8c2f5871a71fcf5e1090c8a032
```

Comment:
@KarolKaczmarek Done. Can I suggest that this is added to .gitignore file in all repositories?